Makefile

```diff
@@ -303,7 +303,7 @@ e2e:
 	$(KUTTL) test
 
 .PHONY: prepare-e2e
-prepare-e2e: kuttl start-kind cert-manager set-test-image-vars docker-build load-image-operator deploy
+prepare-e2e: kuttl start-kind cert-manager deploy-minio set-test-image-vars docker-build load-image-operator deploy
```

Comment:
` deploy-minio`

Does not seem to be related to this PR. Consider moving it to a separate PR.

Reply:
I think he merged #151 here. So, these changes will not be shown in this PR when #151 is merged.

Reply:
Yep, I had to merge the e2e-fixes from #151 into this branch to make the tests pass. It's been merged but somehow these changes are showing up here (maybe because the squash created a new git commit?), anyway, I'll rebase it.